examples/circuit-relaying/README.md

```diff
@@ -0,0 +1,349 @@
+
+Remember the caveat above `Currently a Relay will only work if it already has a connection to the STOP node`? This means that we need to connect our browser nodes to the relay node first.
+
+Having both browsers running side by side (as shown in the first screenshot), enter the `/ip4/127.0.0.1/tcp/4003/ws/ipfs/...` address noted above into the `Connect to Peer` field and hit the `Connect` button:
+
+![](./img/img3.png)
+
+After connecting to the IPFS node, we should see the relay peer show up under the `Peers Connected` box.
+
+![](./img/img4.png)
+
+Let's repeat the same steps with the second tab. Now, both of our browser nodes should be connected to the relay and we can move on to the next step.
+
+### 4. Dial the two browser nodes using a `/p2p-circuit` address
+
+Now that our browsers are both connected to the relay peer, let's get them connected to each other. Head out to the `Addresses` box in one of the tabs, copy the `/p2p-circuit` address and then paste it into the `Connect to Peer` box in the other tab. Repeat these steps on the second tab.
+
+![](./img/img5.png)
+
+Let's hit the `Connect` button on each of the tabs and we should get the two browsers connected and join the chat room.
+
+![](./img/img6.png)
+
+### 5. Send data browser to browser.
+
+Now that we have the two browsers connected, let's try the app out. Type a few words in one of the browser windows and you should see them appear in the other as well!
+
+![](./img/img7.png)
+
+Thats it!
+
+### So what just happened?
+
+Good question! 
+
+- We used [js-ipfs](htpps://github.com/ipfs/js-ipfs) running in the browser with circuit relay enabled:
+  - _Notice the `EXPERIMENTAL.relay.enabled` below_
+
+you can find it in [src/app.js](src/app.js)
+```js
+const ipfs = new IPFS({
+  repo: repo(),
+  EXPERIMENTAL: {
+    relay: {
+      enabled: true,
+      hop: {
+        enabled: true
+      }
+    }
+  },
+  config: {
+    Bootstrap: []
+  }
+})
+```
+
+- We connected the browser nodes to an external node over its websocket transport using the `/ip4/127.0.0.1/tcp/4003/ws/ipfs/...` multiaddr. That external node happens to be a `HOP` node, meaning that it can relay connections for our browsers (and other nodes) allowing them to connect
+
+- And finally we connected the two browser nodes using the `/p2p-circuit/ipfs/...` multiaddr. Take a look at the code below in [src/app.js](src/app.js#L102...L107) - lines 102-107
+
+```js
+    ipfs.swarm.connect(peer, (err) => {
+      if (err) {
+        return console.error(err)
+      }
+      $pAddrs.innerHTML += `<li>${peer.trim()}</li>`
+    })
+```
+
+The above code snippet, handles connecting to other nodes using `ipfs.swarm.connect`, notice how there wasn't anything special we had to do to use the circuit once we had everything connected, all the magic is in the multiaddr! [Multiaddrs](https://multiformats.io/multiaddr/) are **AWESOME**!
```

Comment:
The commas are a little odd in this first sentence. Maybe break it in two and use a semicolon before “all the magic”:

> The above code snippet handles connecting to other nodes using `ipfs.swarm.connect`. Notice how there wasn't anything special we had to do to use the circuit once we had everything connected; all the magic is in the multiaddr! [Multiaddrs](https://multiformats.io/multiaddr/) are **AWESOME**!